canvas_modules/common-canvas/src/common-properties/controls/abstract-table.scss

```diff
@@ -16,6 +16,10 @@
 
 .properties-at-buttons-container {
 	float: right;
+	.bx--btn--ghost.bx--btn--sm {
```

Comment:
Where did you see an issue with this.  This really shouldn't be needed.

Reply:
Default carbon padding in `.bx--btn--ghost.bx--btn--sm` is increased from 12px to 16px. Because of this, `Remove` and `Add column` buttons are not in same line when editor_size is small. 

Carbon version 10.29.0 (16px padding) - 
![Screen Shot 2021-02-25 at 10 51 51 PM](https://user-images.githubusercontent.com/25124000/109266209-960e2b00-77bc-11eb-931e-a93dfd88fcd9.png)

Carbon version 10.20.0 (12px padding) -
![Screen Shot 2021-02-25 at 10 59 20 PM](https://user-images.githubusercontent.com/25124000/109266676-33695f00-77bd-11eb-9187-8c75b20a68b7.png)

Reply:
What does this do in other languages?

Reply:
- Keeping carbon default padding (16px) -

Add & Remove buttons are in same line for - `ja, zh-cn, zh-cw ` locales
Add & Remove buttons are in separate lines for - `en, de, eo, es, fr, it, pt-br, ru` locales

- Override carbon default padding with - `padding: 0 12px` -

Add & Remove buttons are in same line for - `en, ja, zh-cn, zh-cw ` locales
![Screen Shot 2021-02-26 at 11 05 19 AM](https://user-images.githubusercontent.com/25124000/109343675-917c5d80-7822-11eb-8d95-45d54a5c5258.png)

Add & Remove buttons are in separate lines for - `de, eo, es, fr, it, pt-br, ru` locales
![Screen Shot 2021-02-26 at 10 53 13 AM](https://user-images.githubusercontent.com/25124000/109343722-a0631000-7822-11eb-9e5d-56c33f8213ca.png)

![Screen Shot 2021-02-26 at 10 56 56 AM](https://user-images.githubusercontent.com/25124000/109343748-a953e180-7822-11eb-963e-41449143aa6a.png)